docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,1137 @@
+{
```

Comment:
<p>I think it makes sense to substitute <code>alpha</code> in the right side of <code>f</code> since, down below, you use the <code>alpha</code> representation of <code>f</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>